translator/src/main/java/com/google/devtools/j2objc/Options.java

```diff
@@ -94,6 +94,7 @@
   private boolean nullability = false;
   private EnumSet<LintOption> lintOptions = EnumSet.noneOf(LintOption.class);
   private boolean includeGeneratedSources = false;
+  private boolean packagePrefixedFilenames = false;
```

Comment:
Instead of an extra boolean field, use outputStyle and add an enum value to OutputStyleOptions.